From 3dc5a149a2edf5fd5600c8cf8b443c6587d5f1b8 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Sun, 15 Jan 2017 20:29:36 +0100
Subject: [PATCH 7/7] Remove outdated SGML comments

These have not been true for a long time, just get rid of these
comments. Also making easier to conversion to DocBook.
---
 policy.sgml | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/policy.sgml b/policy.sgml
index 0fb3067..1b86b42 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -2375,7 +2375,6 @@ endif
 	</sect1>
       </sect>
 
-<!-- FIXME: section pkg-srcsubstvars is the same as substvars -->
       <sect id="substvars">
 	<heading>Variable substitutions: <file>debian/substvars</file></heading>
 
@@ -4488,12 +4487,6 @@ fi
 		It is an error for a package to contain files which
 		are on the system in another package, unless
 		<tt>Replaces</tt> is used (see <ref id="replaces">).
-		<!--
-		The following paragraph is not currently the case:
-		Currently the <tt>- - force-overwrite</tt> flag is
-		enabled, downgrading it to a warning, but this may not
-		always be the case.
-		-->
 	      </p>
 
 	      <p>
-- 
2.11.0

